src/public/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    <script src="/socket.io/socket.io.js"></script>
+    <script>
+        var socket = io.connect('http://localhost:3000');
+
+        socket.emit('connect_user', 198);
+        socket.on('trip_request_event', function (data) {
+            console.log(data);
```

Comment:
Good progress. but do we need to console the data?

Reply:
Yes, we do need that for printing a received notification data.